src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,64 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final String NEW_LINE = System.lineSeparator();
+    private static final int OPERATION_TYPE_INDEX = 0;
+    private static final int AMOUNT_INDEX = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        createReport(fromFileName, toFileName);
```

Comment:
```suggestion
        List<String> fileContent = readFile(fromFileName);
        String report = createReport(fileContent );
        writeFile(toFileName, report);
```
![image](https://github.com/mate-academy/jv-work-with-file/assets/20326858/a649d187-f6f5-4e1f-9023-e82f20dc99ff)